src/open_inwoner/components/templates/components/Action/ActionStatusButton.html

```diff
@@ -1,10 +1,10 @@
 {% load i18n form_tags icon_tags button_tags dropdown_tags action_tags link_tags %}
 
 <div id="{{ swap_target_id }}">
-{% dropdown class=class text=text text_icon=icon %}
+{% dropdown class=class text=text %}
     {% for status in choices %}
     <div class="dropdown__item">
-        {% button text_icon=None icon=status.icon bordered=False text=status.text class=status.class disabled=status.disabled extra_attributes=status.extra_attributes size="small" type="button" %}
+        {% button text_icon=None bordered=False text=status.text class=status.class disabled=status.disabled extra_attributes=status.extra_attributes size="small" type="button" %}
```

Comment:
I'm unsure if we need to stick to the Dropdown with the 'protrusions' since the opened state is not shown in the design, but let's keep it for now. I do feel like there is way too little spacing here, as compared to the main navigation dropdowns. It could use some designers love.
We use this old dropdown in Contacts and Files also, but those are all from the 'old' designs.
I have a design-meeting tomorrow. If the design changes that would have consequences here.